.github/workflows/pdm-lock-automation.yaml

```diff
@@ -21,6 +21,7 @@ on:
 
 jobs:
   check_pyproject_changes:
+    name: ${{ matrix.directory }} PDM lock update
```

Comment:
@chandrasekharan-zipstack is this the correct place to add this? How have we tested this?

Reply:
Will a manual trigger help to check this?

Reply:
Triggered manually to check this and also updated the PR description @ritwik-g 
https://github.com/Zipstack/unstract/actions/runs/10611169561